Formula/tnftpd.rb

```diff
@@ -0,0 +1,44 @@
+class Tnftpd < Formula
+  desc "NetBSD's FTP server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftpd/lukemftpd-51.tar.gz"
+  sha256 "969b8a35fabcc82759da2433973b9606b7b62f73527e76ac8f18d0a19f473c2a"
+
+  keg_only :provided_pre_high_sierra
+
+  depends_on :xcode => :build
+
+  def install
+    system "tar", "zxvf", "tnftpd-20100324.tar.gz"
+    Dir.chdir "tnftpd-20100324"
+
+    system "./configure"
+    system "make"
+
+    bin.install "src/tnftpd" => "ftpd"
+    man8.install "src/tnftpd.man" => "ftpd.8"
+    man5.install "src/ftpusers.man" => "ftpusers.5"
+    man5.install "src/ftpd.conf.man" => "ftpd.conf.5"
+    etc.install "examples/ftpd.conf"
+    etc.install "examples/ftpusers"
+  end
+
+  def caveats
+    <<-EOS.undent
```

Comment:
This will need changing to the new `<~EOS` style.

Reply:
@DomT4 Ok.